### ISSUE TITLE
eBook clean up recommendations 

### ISSUE THREAD

**Issue by Rick-Anderson** [dotnet/docs#23635]
If this document is preview and the book is finished, I recommend pointing that out up front and maybe deleting most of the content. If they're the same, this should be cleaned up. How do they differ, that should be stated.

PDF link at the top.
`>` Non-standard authors - wastes too much space, too wordy

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: b041c8c5-5a24-a4c6-4e81-baf2f657250f
* Version Independent ID: bd48cc81-9a93-3e33-e196-a8353cefe7ee
* Content: [Porting Existing ASP.NET Apps to .NET Core](https://docs.microsoft.com/en-us/dotnet/architecture/porting-existing-aspnet-apps/)
* Content Source: [docs/architecture/porting-existing-aspnet-apps/index.md](https://github.com/dotnet/docs/blob/main/docs/architecture/porting-existing-aspnet-apps/index.md)
* Product: **dotnet-architecture**
* GitHub Login: @ardalis
* Microsoft Alias: **nanil**

**Comment by ardalis** [comment]
I think @nishanil is planning on removing the Preview notice and ensuring the PDF is updated soon. I'm not sure what content you're suggesting should be cleaned up or deleted. Although eBooks are decomposed into and built from individual markdown files that doesn't mean each individual markdown file is meant to act as a documentation article. It's part of a book and should be considered in that context.

**Comment by erjain** [comment]
The issue can be closed now.
